pkg/cmd/cli/cmd/login.go

```diff
@@ -178,19 +181,71 @@ func (o LoginOptions) Validate(args []string, serverFlag string) error {
 	return nil
 }
 
-// RunLogin contains all the necessary functionality for the OpenShift cli login command
-func RunLogin(cmd *cobra.Command, options *LoginOptions) error {
-	if err := options.GatherInfo(); err != nil {
+func (o *LoginOptions) RunLogin() error {
+	if !o.Force && o.isLikelyAdminKubeConfig() {
```

Comment:
> that's going to break a lot of docs on origin :-/

I don't think we'll regret it.  The number of "I can't do X and I my admin.kubeconfig doesn't work anymore" questions is high.